### DISCUSSION TITLE
v62.0.0 : Unexpected top-level property "languageOptions"

### DISCUSSION THREAD

**Discussion by vlo-rte** [standard/eslint-config-standard-with-typescript#1713] [Q&A] [answered]
Hello,

We have this error when we are tryin to build our project, since the upgrade to version 62.0.0 : 

Error: ESLint configuration in .eslintrc.json » eslint-config-love is invalid:
	- Unexpected top-level property "languageOptions".

Do you know what we have to do to solve that please ? 
Thanks for your help.

Here is our file .eslintrc.json : 
```
{
    "env": {
        "es2021": true,
        "node": true
    },
    "extends": ["love", "plugin:prettier/recommended"],
    "parserOptions": {
        "ecmaVersion": "latest",
        "sourceType": "module"
    },

    "ignorePatterns": ["src/common/**/*"],
    "rules": {
        "@typescript-eslint/consistent-type-imports": "off"
    }
}
```

**Comment by dv-thibeauv** [comment, marked as answer]
You have to use [the new ESLint flat configuration format](https://eslint.org/docs/latest/use/configure/configuration-files)

**Comment by vlo-rte** [comment]
@thibeauv53 it seems to work ! Thanks for your help.
